sycl/cmake/modules/AddSYCLUnitTest.cmake

```diff
@@ -46,58 +46,3 @@ macro(add_sycl_unittest test_dirname link_variant)
     )
   endif()
 endmacro()
-
-macro(add_sycl_unittest_with_device test_dirname link_variant)
```

Comment:
I think it's still used in some `kernel-and-program` tests.

Reply:
Not anymore, these are the last ones.